Tasks/Common/nuget-task-common/NuGetXmlHelper.ts

```diff
@@ -0,0 +1,128 @@
+import * as fs from "fs";
```

Comment:
Good idea, I've added a test to validate NuGetXmlHelper adds the xml elements as expected.